SourceryTests/Parsing/FileParserSpec.swift

```diff
@@ -98,17 +99,6 @@ class FileParserSpec: QuickSpec {
                             ]))
                         }
 
-                        it("extracts method with closure parameter") {
```

Comment:
I guess you missed the PR for this comment, but anyways I've added some KVC tests here :)